config/jobs/kubernetes-sigs/cluster-api-provider-aws/OWNERS

```diff
@@ -1,20 +1,21 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-- dthorsen
-- pydctw
-- shivi28
 - AverageMarcus
+- luthermonson
+- cnmcavoy
+- nrb
+- faiq
```

Comment:
Remove until @faiq can ask for membership?

Reply:
Thanks for the reminder I just asked for membership here: https://github.com/kubernetes/org/issues/4031

Reply:
When this merges https://github.com/kubernetes/org/issues/4733 lets get this change in